src/components/ui/CoralTab.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component,  PropTypes } from 'react';
+import Radium from 'radium';
+
+/**
+ * Tab
+ *
+ * Material Design Reference: https://github.com/google/material-design-lite/tree/mdl-1.x/src/tabs
+ */
+@Radium
+export default class CoralTab extends Component {
+
+  static propTypes = {
+    active: PropTypes.bool,
+    className: PropTypes.string,
+    onClick: PropTypes.func,
+    tabId: PropTypes.number,
+    style: PropTypes.object
+  };
+
+  render() {
+    const {
+      active,
+      className,
```

Comment:
does this improve visibility or reduce it? we can move this to just a couple of lines IMO